src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -106,18 +128,36 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
 
   public slideDirection: string;
 
+  public set tabIndex(value: number) {
+    this._tabIndex = value;
+  }
+
+  public get tabIndex(): number {
+    return this._tabIndex;
+  }
+
+  @ViewChild('skyRepeaterItem')
+  public itemRef: ElementRef;
+
+  private focusableChildren: HTMLElement[] = [];
```

Comment:
Since Angular components are meant to be platform agnostic, we shouldn't put any references to the HTML API in the component (that's what our DOM "adapters" are for). Ideally, any interactions with "native elements" would be done in the adapter service. Would it be possible to offload the tabbing and active-index functionality to the adapter?

Reply:
If the above suggestion isn't possible, this is one of the rare instances where `any` is appropriate:
```
private focusableChildren: any[];
```

Reply:
Yeah totally. That was an oversight. Thanks for catching that!

Reply:
Done. Replaced with a call to the adapter's `getFocusableChildren()` method.